### ISSUE TITLE
"Initial" CollectionView render required in 2.2+?

### ISSUE THREAD

**Issue by mrliptontea** [marionettejs/backbone.marionette#2016]
Hi,

I wanted to upgrade Marionette, previously I was using version 2.1.0.
Now, in 2.2.2 (and 2.2.0) this issue occured, best explained with code:

```javascript
var employees      = new App.Collections.Employee();
var employees_list = new App.Views.EmployeeList({ collection: employees });

employees.reset(App.data.collection); // employees_list rendered automatically after this in 2.1.0

// after upgrade to 2.2.0 it does nothing until I add "initial" render:
employees_list.render();
```

Additionaly, Employee collection is PageableCollection and I use paginator. After first, initial render, the collection renders as it should after reset, but nothing happens if I ommit this first render.
Is this intended? From what I see in Marionette's code it is, but change log or docs don't mention it.

**Comment by samccone** [comment]
closing, sorry about any problems this caused you @mrliptontea this was our bad :(

**Comment by mrliptontea** [comment]
@samccone of course, but I think the documentation still lacks the information that you should render "manually" first to bind automatic rendering.

**Comment by jasonLaster** [comment]
Alright, lets make sure that our CV docs show the need for a `render` call

**Comment by ahumphreys87** [comment]
fixed with #2499

**Comment by mrliptontea** [comment]
:+1:
